### PR TITLE
UI specifications and iframe documentation

### PR DESCRIPTION
**Pull request by xiexr151e**
This PR deals with 2 things:
1. Milind's UI specifications on the GIS button
2. Changing the documentation on the iframe to point to GIS website

### CODE REVIEW

File: src/utils/gaa.js [hunk 1]
```diff
@@ -157,7 +157,7 @@ const REGWALL_HTML = `
   .gaa-metering-regwall--iframe {
     border: none !important;
     display: block !important;
-    height: 36px !important;
+    height: 44px !important;
```
Comment (slmartinet) [inline src/utils/gaa.js:160]: Out of curiosity, why this change in height please?

**Comment by slmartinet** [comment]
Looks good to me. Thank you

File: src/utils/gaa.js (hunk 1 above)
Comment (xiexr151e) [reply src/utils/gaa.js:160]: This is because the default height for the button is 44px.
I was previously having an issue in which the bottom edge of the button was being cut off. Changing this height resolved the issue.

**Review by adolgachev — APPROVED** [review]
